test/python/scenarios.json

```diff
@@ -13,7 +14,8 @@
         "features": {
             "python": {
                 "version": "none",
-                "installTools": true
+                "installTools": true,
+                "skipVulnerabilityPatching": false
```

Comment:
Can you add another test to validate `skipVulnerabilityPatching` where value is "true"? This should contain the vulnerable packages.

Reply:
done

Reply:
I don't see another scenario added to `test/python/scenarios.json` with ` "skipVulnerabilityPatching": true` 🤔 

I just want to make sure that users have the ability to disable this patch, the condition should do it. However, good to have a test for validation, in case it breaks in future.

Reply:
have added this test scenario